content/04.experiment-types.md

```diff
@@ -16,4 +16,80 @@ Proteomics experiments can be both qualitative and quantitative.
+
+4. protein-painting [@doi:10.1038/ncomms5413; @doi:10.1016/S0959-440X(02)00283-X]
+Protein painting is an optimized MS-based technique in which first the samples are coated with paint molecules. These paint molecules will coat the surfact but will not have access to the protein-protein interface regions that solvents cannot get to. The parts that are painted will not be subjected to trypsin cleavage while the unpainted areas will. After proteolysis, the peptides samples will be subjected to MS. The samples will only show the unmodified opposing regions in the protein-protein interaction at the time of the painting. 
+
+5. LiP-MS (limited proteolysis mass spectrometry) [@doi:10.1016/j.tibs.2020.05.006]
+LiP-MS is a two-step process that begins with the separation of proteins of interest using a high-performance liquid chromatography (HPLC) system. The separated proteins are then subjected to a liquid-phase micro-sampling (LPMS) procedure, which allows for the analysis of a small number of proteins at a time. This methodology allows for the identification and quantification of the proteins of interest without the need for sample pre-treatment or proteomic analysis.
+
+6. radical footprinting 
+This technique uses hydroxyl radical footprinting and MS in order to elucidate protein structures, assembly, and interactions within a large macromolecule. [@doi:10.1002/9781118970195.ch15] Over the years, various approaches to makes hydroxide radicals has been investigated for biological footprinting studies in nucleic acid/ligand interactions. [@doi:10.1016/j.cbpa.2005.02.009; @doi:10.1002/0471142700.nc0607s07; @doi:10.1093/nar/gkz089] Below are known methods of producing radicals.
+
+1. Fenton and Fenton‐like Chemistry
+2. Electron-Pulse Radiolysis
+3. High‐Voltage Electrical Discharge
+4. Synchrotron X‐ray Radiolysis of Water
+5. Plasma Formation of OH Radicals
+6. Photolysis of Hydrogen Peroxide
```

Comment:
can we have a citation example for each method?

Reply:
It is there now

Reply:
AWESOME!